packages/backend/src/app/workers/sandbox/provisioner/sandbox-cache-key.ts

```diff
@@ -38,8 +38,8 @@ export const extractProvisionCacheKey = (params: ProvisionCacheInfo): string =>
 
 
 
-const extractCodeCacheKey = ({ artifactSourceId }: CodeProvisionCacheInfo): string => {
-    return `CODE-artifactSourceId-${artifactSourceId}`
+const extractCodeCacheKey = ({ sourceCodeHash }: CodeProvisionCacheInfo): string => {
+    return `CODE-artifactSourceId-${sourceCodeHash}`
```

Comment:
please note that `CODE-artifactSourceId-` needs to be updated to `CODE-sourceCodeHash-`.